lib/rbUtil.js

```diff
@@ -225,7 +225,7 @@ rbUtil.makeLogger = function(conf) {
     function bindAndChild (logger) {
         var log = logger.log.bind(logger);
         log.child = function(args) {
-            return bindAndChild(new Logger(null, logger.logger.child(args)));
+            return bindAndChild(new Logger(conf, logger.logger.child(args)));
```

Comment:
This ensures that child loggers inherit the same log level as the parent, rather than defaulting to `warn`.